src/vs/workbench/parts/terminal/electron-browser/terminalLinkHandler.ts

```diff
@@ -188,12 +238,65 @@ export class TerminalLinkHandler {
 			return TPromise.as(void 0);
 		}
 
+		const linkUrl = this.extractLinkUrl(link);
 		// Open an editor if the path exists
-		return pfs.fileExists(link).then(isFile => {
+		return pfs.fileExists(linkUrl).then(isFile => {
 			if (!isFile) {
 				return null;
 			}
 			return link;
 		});
 	}
+
+	/**
+	 * Returns line and column number of URl if that is present.
+	 *
+	 * @param link Url link which may contain line and column number.
+	 */
+	public extractLineColumnInfo(link: string): LineColumnInfo {
+		const matches: string[] = this._localLinkRegex.exec(link);
+
+		let lineColumnClauseLength = this._lineAndColumnClauses.length;
+		let lineColumnInfo: LineColumnInfo = {};
+
+		let lineAndColumnMatchIndex = this._winLineAndColumnMatchIndex;
+
+		if (platform.Platform.Windows !== this._platform) {
```

Comment:
Can you swap the if? Normally we put the constant expression on the right: `this._platform !== platform.Platform.Windows`